### DISCUSSION TITLE
Parabola

### DISCUSSION THREAD

**Discussion by parasti** [Neverball/neverball#331] [Show and tell]
Some years ago I reimplemented most of the Neverball renderer in WebGL. No recompilation of source files, no emulation of OpenGL, just plain old JavaScript that renders SOLs in the browser.

It's useless, because I never finished it, but I got a lot of enjoyment out of just messing around with it and being able to fly and look at the background from behind.

https://github.com/parasti/parabola

**Comment by rlk** [comment]
Fun! Thanks for putting this up.

**Comment by ChallengeSY** [comment]
Opted to play around with this app; got custom inputted levels, backgrounds, and gradients working correctly so far.

I may (and did) integrate some of my custom levels into a locally hosted Parabola.
